relation_engine_server/README.md

```diff
@@ -173,45 +173,102 @@ If you try to update a collection and it fails validation against a JSON schema
 * `"value"` - The (possibly nested) value in your data that failed validation
 * `"path"` - The path into your data where you can find the value that failed validation
 
-### GET /api/v1/data_sources
+### PUT /api/v1/specs/
```

Comment:
moved the `GET /api/v1/data_sources` information to be with the `GET /specs/collections`, etc., endpoints